### ISSUE TITLE
Clear localisation from selected locale

### ISSUE THREAD

**Issue by thetanz-geoff** [tractorcow-farm/silverstripe-fluent#771]
I'm likely overlooking something here so my apologies for the naive question.

I've created 3 locales: en_NZ, es_MX and mi_NZ. I've added an item and provided English and Spanish localisations. As a result, Māori correctly shows as "Not localised":
![image](https://user-images.githubusercontent.com/2903322/212496187-5a1f4963-893f-4af3-87f9-346259b03c93.png)

I can now use "Copy from" or "Copy to" to populate Māori from English by default, and begin to introduce localisations.

If I decide that I no longer want to provide Māori localisations for this item, and would like to revert to the previous state (fallback to English), how can I do so? I see from the menu that I can remove localisations from "all except 'Maori (New Zealand)'", but no clear way to remove localisations for Māori only.
![image](https://user-images.githubusercontent.com/2903322/212496552-f71cd620-3815-48e4-9cd9-5209c7505c2e.png)

Basically, I want to be able to revert to that "Not localised" state. Please let me know whether I'm missing something.

**Comment by thetanz-geoff** [comment]
"Unpublish and archive" does the trick while leaving the existing item intact.
Hence, "Unpublish and archive (all locales)" becomes the new way to delete the full item and all its localised versions.
